ui/app/src/components/SiteSearchFilters/SiteSearchFilters.tsx

```diff
@@ -169,6 +169,53 @@ export default function SiteSearchFilters(props: SiteSearchFiltersProps) {
     }
   };
 
+  const filterToFacet = (filterKey, filterValue) => {
```

Comment:
Couldn't find any closures on this function, in which case we can move out of the component body.